React/Views/RCTRefreshControl.m

```diff
@@ -37,7 +74,13 @@ - (void)setRefreshing:(BOOL)refreshing
 {
   if (super.refreshing != refreshing) {
```

Comment:
Probably no need to use super here, since you aren't overriding isRefreshing. Maybe it's safer though.